adding_species_manuscript.tex

```diff
@@ -205,8 +205,8 @@ \section*{Introduction}
 such as inbreeding or selfing \citep{Blischak2020}, skewed offspring
 distributions \citep{Montano2016}, and intense artificial selection \citep{MacLeod2013, MacLeod2014}.
 
-Simulations can be useful at all stages of this work --
-for planning studies, analyzing data, testing inference methods,
+Simulations can be useful at all stages of this work---for
```

Comment:
Why three dashes here? Is this a latex thing?

Reply:
An em-dash is three hyphens, `---`, while just two gives an en-dash. https://www.scribbr.com/language-rules/dashes/